macros/all_passthrough_column_check.sql

```diff
@@ -0,0 +1,10 @@
+{% macro all_passthrough_column_check(relation, get_columns) %}
+
+{% set available_passthrough_columns = fivetran_utils.remove_prefix_from_columns(
+                columns=adapter.get_columns_in_relation(ref(relation)), 
+                prefix='property_', exclude=get_macro_columns(get_columns))
+            %}
```

Comment:
```suggestion
{% set available_passthrough_columns = fivetran_utils.remove_prefix_from_columns(
                columns=adapter.get_columns_in_relation(ref(relation)), 
                prefix='property_', exclude=get_macro_columns(get_columns)) | length
            %}
```

Reply:
@fivetran-jamie I realized in my local version I had an uncommitted change. Please see this suggested edit. This is required to actual infer the length of the result of avaiable_passthrough_columns. If this is not included the conditional in the staging macro will never be met.